fetcher/fetcher.go

```diff
@@ -81,6 +81,7 @@ func (c *Fetcher) workInit() {
 	c.worker.PushIf("service_bindings", c.fetchServiceBindings, filters.ServiceBindings)
 	c.worker.PushIf("users", c.fetchUsers, filters.Events)
 	c.worker.PushIf("events", c.fetchEvents, filters.Events)
+	c.worker.PushIf("application_buildpack", c.fetchApplications, filters.Applications)
```

Comment:
not necessary, fetchApplications has already been pushed to workers at line 63

Reply:
Fixed